apollo-router/src/plugins/authentication/mod.rs

```diff
@@ -150,6 +98,13 @@ struct JWTConf {
     /// Alternative sources to extract the JWT
     #[serde(default)]
     sources: Vec<Source>,
+    /// Control the behavior when an error occurs during the authentication process.
+    ///
+    /// Defaults to `Error`. When set to `Continue`, requests that fail JWT authentication will
+    /// continue to be processed by the router, but without the JWT claims in the context. When set
+    /// to `Error`, requests that fail JWT authentication will be rejected with a HTTP 403 error.
+    #[serde(default = "OnError::default")]
```

Comment:
If JWTConf and OnError implementsDefault you don't need this
```suggestion
```
Same for the other fields